### PR TITLE
Add Ableton Live Lite

### PR DESCRIPTION
**Pull request by panagiotisTB**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused. ([closed](https://github.com/caskroom/homebrew-cask/pull/16064), but not sure if refused)
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free. (audit for ableton-live-lite: passed)
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

PS: It's my first contribution to a github repo. Sorry if I got something wrong. Always open for constructive criticism.

### CODE REVIEW

**Comment by ptbobolakis** [comment]
Not sure why checks failed. 
Maybe because I have macOS 10.11 w/ Xcode 7.3 installed.
`➜  ~ ruby -v`
`ruby 2.0.0p648 (2015-12-16 revision 53162) [universal.x86_64-darwin15]`

I have no idea. Any help would be appreciated.

File: Casks/ableton-live-lite.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  else
+    sha256 '271e5a7167338ddfdc8abd15abf40d02d218137f013df9d8d30267847f5e5c4c'
+    url "http://cdn2-downloads.ableton.com/channels/#{version}/ableton_live_lite_#{version}_64.dmg"
+  end
+
+  name 'Ableton Live Lite'
+  homepage 'https://www.ableton.com/live-lite/'
+  license :commercial
+
+  app "Ableton Live #{version.to_i} Lite.app"
```
Comment (vitorgalvao) [inline Casks/ableton-live-lite.rb]: Do `#{version.major}` instead.

File: Casks/ableton-live-lite.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  zap delete: '~/Library/*/*[Aa]bleton*',
+      rmdir:  '~/Music/Ableton/Factory Packs'
+  #:trash => '~/Music/Ableton/User Library'
+end
```
Comment (vitorgalvao) [inline Casks/ableton-live-lite.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

**Comment by vitorgalvao** [comment]
> ([closed](https://github.com/caskroom/homebrew-cask/pull/16064), but not sure if refused)

What you should take from that discussion is that we’ll only accept this cask in [caskroom/versions](https://github.com/caskroom/homebrew-versions). Please submit it there.

The failing build seems to be mostly Travis messing up.

**Pull request closed without merging**